package-examples/ghost/package-context.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kptfile.kpt.dev
```

Comment:
Both `kpt pkg init` and `kpt pkg get --for-deployment` generate the package-context.yaml. `kpt pkg get` won't.

This package-contex.yaml is generated by `kpt pkg init`, which is how we suggest users to create blueprint from scratch in kpt CLI https://kpt.dev/guides/namespace-provisioning-cli?id=initialize-a-package. @droot shall we change kpt on `kpt pkg init` to not creating `package-context.yaml`? But then it will break the local testing. Or we can add one more step to delete this files manually.

Reply:
@bgrant0607 your understanding is correct and current behavior is inline with that.

@yuwenma . No, we don't have to change the `kpt pkg init` behavior. 

Listing the scenario that we need to confirm:

kpt pkg get <ghost-super-pkg> app-1 --for-deployment

1. the app-1/package-context.yaml will contain `data.name: app-1`
2. app-1/ghost/package-context.yaml  ??
3. app-2/mariadb/package-context.yaml ??

And when you run `kpt fn render app-1`, the set-namespace will use the package-context at the top-level, so everything should be fine, but we need to confirm this.

Reply:
I added the full steps in README.yaml and approved this works well:

`kpt pkg get --for-deployment` will create package-context.yaml under each sub directory as well as the parent directory, with the directory name as the `data.name`, which are `ghost-app`, `mariadb` `root-dir`. Running set-namespace on root updates all namespaces to the value in `root-dir` package context `data.name`. Then, `kpt live apply root-dir` deploys all resources to the cluster successfully